aten/src/ATen/native/sparse/SparseCsrTensor.cpp

```diff
@@ -42,6 +43,8 @@
 #include <ATen/ops/resize_native.h>
 #include <ATen/ops/row_indices_native.h>
 #include <ATen/ops/select_native.h>
+#include <ATen/ops/select_copy.h>
+#include <ATen/ops/select_copy_native.h>
```

Comment:
nit: check whether native version is used at all.

Reply:
Without `select_copy_native.h`, I got a linker error about a missing symbol. So I concluded that it is needed.

Reply:
You can resolve that by using `at::select_copy` directly. Looks like the code execution takes place in the `at::native` namespace.

Reply:
`select_copy` is function-only and therefore I am already using `at::select_copy`:
https://github.com/pytorch/pytorch/blob/1abc4b5e2e6659e22cfb1ab0ae9be589ea06a79d/aten/src/ATen/native/sparse/SparseCsrTensor.cpp#L776

The linker error was about missing `select_copy_sparse_csr` symbol.